eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -393,26 +393,48 @@ function GetExistingTags($apiUrl) {
   }
 }
 
-# Retrieve release tag for artiface package. If multiple packages, then output the first one.
-function RetrieveReleaseTag($pkgRepository, $artifactLocation, $continueOnError = $true) {
+# Retrieve package info for artiface package.
+function RetrieveReleasePackageInfo($pkgRepository, $artifactLocation, $continueOnError = $true) {
```

Comment:
Lets avoid making this change and instead use the functions directly in the language-settings for our work here.